tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -244,4 +364,26 @@ function ( $preempt, $args, $url ) use ( $request_url, $response_data, $response
 			3
 		);
 	}
+
+	/**
+	 * Adds a 'pre_http_request' filter that will ensure the request for the
+	 * given URL will return WP_Error.
+	 *
+	 * @param string $request_url   Request URL to modify response for.
+	 * @param (WP_Error)  $response_error Response WP_Error to return.
```

Comment:
This shouldn't be in parentheses, and the parameter docs should be aligned.
```suggestion
	 * @param string   $request_url    Request URL to modify response for.
	 * @param WP_Error $response_error Response WP_Error to return.
```